apps/web/src/components/Common/Layout.tsx

```diff
@@ -48,23 +48,21 @@ const Layout: FC<LayoutProps> = ({ children }) => {
 
   const { id: sessionProfileId } = getCurrentSession();
 
-  const logout = (reload = false) => {
+  const logout = () => {
     resetPreferences();
     resetFeatureFlags();
     resetPro();
```

Comment:
I guess we can move all these inside signout method itself so we can just call signout everywhere!